devito/arch/compiler.py

```diff
@@ -468,11 +469,6 @@ class ArmCompiler(GNUCompiler):
 
     def __init_finalize__(self, **kwargs):
```

Comment:
this constructor can go

Reply:
The whole class is redundant now...but keeping for compatibility I guess.